.github/workflows/bolt-test.yaml

```diff
@@ -37,6 +37,10 @@ jobs:
       - name: Bolt Test
         run: nix-shell --pure --run "bolt test"
 
+      - name: Tmate
+        if: failure()
+        uses: mxschmitt/action-tmate@v3
```

Comment:
This is pretty dope, do we care about security at all?

https://github.com/mxschmitt/action-tmate?tab=readme-ov-file#use-registered-public-ssh-keys